src/config.c

```diff
@@ -229,8 +229,6 @@ typedef union typeData {
 typedef struct typeInterface {
     /* Called on server start, to init the server with default value */
     void (*init)(typeData data);
-    /* Called on server start, should return 1 on success, 0 on error and should set err */
-    int (*load)(typeData data, sds *argc, int argv, const char **err);
```

Comment:
`.load` is never set and never called, so I agree, it's safe to remove. Do you know which function is actually called on server start to load the data? I assume it is either init or get. It would be good to update the comment at the function which is used.

Reply:
It's set I'm looking at this now to see if there is any other dead code related to configs.